### PR TITLE
Added logic for http DELETE method to delete tasks

### PR DESCRIPTION
**Pull request by niemepet1**

### CODE REVIEW

File: src/main/java/fi/bilot/resources/TaskResource.java [hunk 1]
```diff
@@ -69,22 +69,60 @@ public JsonObject getTask(@PathParam("id") int id) {
...
+        taskList.createTask("Another task");
+
+        final Task task = taskList.getTask(id);
+
+        if(task != null){
+            taskList.deleteTask(task);
+            final JsonObject responseEntity = Json.createObjectBuilder()
+                    .add("message", "task "+ task.id +" deleted")
+                    .build();
+            return Response.status(200)
```
Comment (SergeTsybaBilot) [inline src/main/java/fi/bilot/resources/TaskResource.java]: You don't need an extra message here. In general, you don't give any response when an expected result has been reached, only when something went wrong.
Additionally, if you change this to a response without message, the status should be different.

File: src/main/java/fi/bilot/resources/TaskResource.java [hunk 2]
```diff
@@ -69,22 +69,60 @@ public JsonObject getTask(@PathParam("id") int id) {
...
+    @DELETE
+    @Path("tasks/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response deleteTask(@PathParam("id") int id){
+        taskList.createTask("Some different task");
+        taskList.createTask("Another task");
+
+        final Task task = taskList.getTask(id);
+
+        if(task != null){
```
Comment (SergeTsybaBilot) [inline src/main/java/fi/bilot/resources/TaskResource.java]: Minor tip here. You're having an inverse logic check here. Namely, you're doing a check
if not null then ..., else if null then ...
More straightforward is to reverse condition, namely
if null then ..., else ...
Hope it makes sense.

**Comment by SergeTsybaBilot** [comment]
Also, there is some commented out code in TaskList tests, uncomment and update those as well.

File: src/main/java/fi/bilot/resources/TaskResource.java (hunk 2 above)
Comment (niemepet1) [inline src/main/java/fi/bilot/resources/TaskResource.java]: Ok, thanks for the tip

**Pull request merged**